src/lib/blueprint.ts

```diff
@@ -117,14 +121,327 @@ interface ObjectProperty extends BaseProperty {
 
 type Method = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
 
+type LowercaseMethod = Lowercase<Method>
+
 export interface TypesModule {
-  openapi: Openapi
+  openapi: OpenAPI
 }
 
 export const createBlueprint = ({ openapi }: TypesModule): Blueprint => {
+  const isFakeData = openapi.info.title === 'Foo'
+  const targetPath = '/acs/systems/list'
+  const targetSchema = 'acs_system'
+
   return {
     name: openapi.info.title,
-    routes: [],
-    resources: {},
+    routes: createRoutes(openapi.paths, isFakeData, targetPath),
+    resources: createResources(
+      openapi.components.schemas,
+      isFakeData,
+      targetSchema,
+    ),
+  }
+}
+
+const createRoutes = (
+  paths: OpenAPIPaths,
+  isFakeData: boolean,
+  targetPath: string,
+): Route[] => {
+  return Object.entries(paths)
+    .filter(([path]) => isFakeData || path === targetPath)
+    .map(([path, pathItem]) => createRoute(path, pathItem))
+}
+
+const createRoute = (path: string, pathItem: OpenAPIPathItem): Route => {
+  const pathParts = path.split('/')
+  const routePath = `/${pathParts.slice(1, -1).join('/')}`
+
+  return {
+    path: routePath,
+    namespace: { path: `/${pathParts[1]}` },
+    endpoints: createEndpoints(path, pathItem),
+    subroutes: [],
+  }
+}
+
+const createEndpoints = (
+  path: string,
+  pathItem: OpenAPIPathItem,
+): Endpoint[] => {
+  return Object.entries(pathItem)
+    .filter(
+      ([, operation]) => typeof operation === 'object' && operation !== null,
+    )
+    .map(([method, operation]) =>
+      createEndpoint(method, operation as OpenAPIOperation, path),
+    )
+}
+
+const createEndpoint = (
+  method: string,
+  operation: OpenAPIOperation,
+  path: string,
+): Endpoint => {
+  const pathParts = path.split('/')
+  const endpointPath = `/${pathParts.slice(1, -1).join('/')}`
+
+  return {
+    name:
+      'operationId' in operation && typeof operation.operationId === 'string'
+        ? operation.operationId
+        : `${path.replace(/\//g, '')}${method.charAt(0).toUpperCase() + method.slice(1).toLowerCase()}`,
+    path: endpointPath,
+    description:
+      'description' in operation && typeof operation.description === 'string'
+        ? operation.description
+        : '',
+    isUndocumented: false,
+    isDeprecated: false,
+    deprecationMessage: '',
+    parameters: createParameters(operation),
+    request: createRequest(method, operation),
+    response: createResponse(
+      'responses' in operation ? operation.responses : {},
+    ),
+  }
+}
+
+const createParameters = (operation: OpenAPIOperation): Parameter[] => {
+  return 'parameters' in operation && Array.isArray(operation.parameters)
+    ? operation.parameters
+        .filter((param) => typeof param === 'object' && param !== null)
+        .map(createParameter)
+    : []
```

Comment:
Avoid ternay if early return is possible
```suggestion
  if ('parameters' in operation && Array.isArray(operation.parameters)) {
    return operation.parameters
        .filter((param) => typeof param === 'object' && param !== null)
        .map(createParameter)
  }
  return []
```